lib/web_console/templates/console.js.erb

```diff
@@ -370,4 +373,113 @@ REPLConsole.prototype.scrollToBottom = function() {
   this.inner.scrollTop = this.inner.scrollHeight;
 };
 
+/**
+ * Install the console into the element with a specific ID.
+ * Example: REPLConsole.installInto("target-id")
+ */
+REPLConsole.installInto = function(id) {
+  var consoleElement = document.getElementById(id);
+  var replConsole = new REPLConsole({
+    promptLabel: consoleElement.dataset.initialPrompt,
+    commandHandle: function(line) {
+      var _this = this;
+      var url = consoleElement.dataset.remotePath;
+      var params = "input=" + encodeURIComponent(line);
+      putRequest(url, params, function(xhr) {
+        var response = JSON.parse(xhr.responseText);
+        _this.writeOutput(response.output);
+      });
+    }
+  });
+
+  replConsole.install(consoleElement);
+};
+
+// Hash object for console events.
+REPLConsole.events = {}
+
+/**
+ * Bind and trigger console events ("changeFrame" and etc...?).
+ */
+function ConsoleEvent() {
```

Comment:
The only thing we use those events for, is to communicate binding changes. Won't it be easier to just expose the current console instance somewhere, say the REPLConsole object (REPLConsole.current) and call the method there?

If we really need events in the future, we can add them then.